lib/exchange_clients/bitfinex/transformers/symbol.js

```diff
@@ -7,7 +7,7 @@ const _includes = require('lodash/includes')
  * @param {string} - symbol
  * @return {Object} - market
  */
-module.exports = (symbolsByContext, sym) => {
+module.exports = (symbolsByContext, sym, lc = new Map()) => {
```

Comment:
no need for a default value here? 

we only have one usage of the function, but if lc is forgotten by accident we should end up with an error for the programmer

currently it silently fails, and everything looks nice at first, but later at runtime elements are missing